src/commands/sourcemaps/inject.rs

```diff
@@ -96,11 +96,15 @@ pub fn execute(matches: &ArgMatches) -> Result<()> {
         extensions.push("mjs");
     }
 
+    // Sourcemaps should be discovered regardless of which JavaScript extensions have been selected.
```

Comment:
Can you refactor lines above to `.unwrap_or_else(|| vec!["js", "cjs", "mjs"]);` while you're at it?